### PR TITLE
Fix passing around mutex by value.

### PR DESCRIPTION
**Pull request by simitt**
Issue found by running go vent.

### CODE REVIEW

**Comment by simitt** [comment]
jenkins, test this please.

**Comment by simitt** [comment]
Great suggestion @roncohen I adapted to use a `map` now.

File: tests/approvals.go [hunk 1]
```diff
@@ -42,24 +41,33 @@ func ApproveJson(received map[string]interface{}, name string, ignored set.Set)
 	return nil
 }
 
-func ignored_key(data *map[string]interface{}, ignored set.Set) {
+func hasKey(coll []string, k string) bool {
```
Comment (jalvz) [inline tests/approvals.go]: you should remove this func now?

**Comment by simitt** [comment]
jenkins, test this please

**Pull request merged**